src/components/calcite-alert/calcite-alert.tsx

```diff
@@ -120,13 +120,17 @@ export class CalciteAlert {
         ref={(el) => (this.closeButton = el)}
         type="button"
       >
-        <calcite-icon icon="x" scale="m" />
+        <calcite-icon icon="x" scale={this.scale === "s" ? "s" : "m"} />
       </button>
     );
     const queueCount = (
-      <div class={`${this.queueLength > 1 ? "active " : ""}alert-queue-count`}>
+      <calcite-chip
+        class={`${this.queueLength > 1 ? "active " : ""}alert-queue-count`}
+        scale={this.scale}
+        value="Amount of alerts"
+      >
         +{this.queueLength > 2 ? this.queueLength - 1 : 1}
-      </div>
+      </calcite-chip>
```

Comment:
This should probably include a test, to make sure the element renders in this scenario.

And can you elaborate on the chip's "Amount of alerts" value?

Reply:
Thanks for taking a look @caripizza.

I don't think I have the technical knowledge for writing tests for this. Should I create an issue for someone who is more technical to help with this?

On the "Amount of Alerts", wasn't really sure what to add here. Should I just add the actual value that renders?

Reply:
No worries, I'll push up the e2e test for this. For the chip value, we can use the chip's text content (the queue count).